Classes/Fusion/Helper/NodeInfoHelper.php

```diff
@@ -95,8 +95,9 @@ public function renderNodeWithMinimalPropertiesAndChildrenInformation(
         $nodeInfo['properties'] = [
             // if we are only rendering the tree state,
             // ensure _isHidden is sent to hidden nodes are correctly shown in the tree.
-            '_hidden' => $node->tags->contain(SubtreeTag::fromString('disabled')),
-            '_hiddenInIndex' => $node->getProperty('_hiddenInIndex'),
+            // TODO: we should export this correctly named, but that needs changes throughout the JS code as well.
+            '_hidden' => $node->tags->withoutInherited()->contain(SubtreeTag::disabled()),
```

Comment:
ah i see nice fix here ;) this is currently broken as nodes are always shown as explicitly hidden